examples/ts/btc/omni/index.ts

```diff
@@ -0,0 +1,112 @@
+/**
+ * Send or create an omni asset from a multi-sig wallet at BitGo.
+ *
+ * Copyright 2024, BitGo, Inc.  All Rights Reserved.
+ */
+import { Wallet } from 'modules/bitgo/src';
+import { omniConfig } from './config';
+import * as superagent from 'superagent';
+import * as utxolib from '@bitgo/utxo-lib';
+
+const RECEIVE_ADDRESS = '';
+const SEND_ADDRESS = '';
+const ASSET_ID = 31;
+
+async function getWallet() {
+  return await omniConfig.sdk.coin(omniConfig.coin).wallets().get({ id: omniConfig.walletId });
+}
+
+/**
+ * Send an omni asset to a receiver. This function is used when you have sent an omni asset to a BitGo BTC wallet
+ * and need to manually recover it
+ * This function assumes that:
+ *   - Your address has a single unspent that is large enough to cover the transaction
+ *   - The receiver address is a legacy address, otherwise the transaction will not be recognized by the omni explorer
```

Comment:
```suggestion
 *   - The receiver address is a legacy or wrapped segwit ([reference](https://developers.bitgo.com/coins/address-types)) address, otherwise the transaction will not be recognized by the omni explorer. 
```